runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiSMREntry.java

```diff
@@ -64,6 +69,8 @@ public void setEntry(ILogData entry) {
 
     @Override
     public List<SMREntry> getSMRUpdates(UUID id) {
+        // FIXME MultiSMREntry should not implement ISMRConsumable; we have MultiObjectSMREntry for that
```

Comment:
Any consumable SMR entry should implement ISMRConsumable, I don't see how MultiSMREntry should be any different!